src/Talis/Persona/Client/Signing.php

```diff
@@ -0,0 +1,120 @@
+<?php
+namespace Talis\Persona\Client;
+
+class Signing extends Base
+{
+    /**
+     * Checks the supplied config, verifies that all required parameters are present and
+     * contain a non null value;
+     *
+     * @param array $config the configuration options to validate
+     * @throws \InvalidArgumentException If the config is invalid
+     */
+    protected function checkConfig(array $config)
```

Comment:
What is this used for? If you want anyone using Signing to be forced to provide an implementation of this then shouldn't this be an abstract class and this stub declared abstract?

I can't see that this method is ever used?  The Token's class still has an implementation of this method but doesn't refer to this Signing Class at all.